tests/console/btrfsmaintenance.pm

```diff
@@ -0,0 +1,88 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check btrfsmaintenance for functionality
+# - run btrfs-balance.sh and btrfs-scrub.sh
+# - Check if btrfsmaintenance-refresh.service is present and started properly
+# - Check if btrfs-scrub is scheduled
+# - Check if btrfs-balance is scheduled
+# - Checks for btrfs-related cron jobs after uninstalling btrfsmaintenance
+# Maintainer: Felix Niederwanger <felix.niederwanger@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+use version_utils "is_sle";
+
+my $btrfsmaintenance_removed = 0;
+
+sub btrfs_service_check {
+    my $service = $_[0];
+    # Check if the given btrfs service (e.g. scrub or balance) is enabled in one of the following methods:
+    # - via btrfsmaintenance-refresh.service
+    # - via a systemd timer service
+    # - or if known as a timer
+    if (script_run("systemctl status btrfsmaintenance-refresh.service | grep -E -v 'uninstall|none' | grep $service") == 0) {
+        record_info("btrfs-$service", "btrfs-$service enabled via btrfsmaintenance-refresh.service");
+        return 0;
+    } elsif (script_run("systemctl status btrfs-$service.timer | grep -iv inactive | grep -i active") == 0) {
```

Comment:
Why not just `grep -i active` ?

Reply:
or just use `systemctl is-active` ?

Reply:
Changed to `is-enabled`:

    if (script_run("systemctl status btrfsmaintenance-refresh.service | grep -E -v 'uninstall|none' | grep $service") == 0) {
        record_info("btrfs-$service", "btrfs-$service enabled via btrfsmaintenance-refresh.service");
        return 0;
    } elsif (script_run("systemctl is-enabled btrfs-$service.timer | grep enabled") == 0) {```